luigi/contrib/azureblob.py

```diff
@@ -164,7 +164,10 @@ def rename_dont_move(self, path, dest):
 
     @staticmethod
     def splitfilepath(filepath):
-        splitpath = filepath.split("/")
+        if ("/" in filepath):
```

Comment:
while this looks good and none of the azureblob tests failed (to my knowledge, would it be possible to use `os.sep` instead of hardcoding the path here?

Reply:
That is definitely a better idea. I also updated the test paths to be separated by os.sep

Reply:
Oops, turns out this doesn't work. When I tested it, splitfilepath is being sent paths that are sometimes '/' separated and sometimes '\' separated.